### PR TITLE
test/STADTPULS 440/load test staging api

### PR DESCRIPTION
**Pull request by ff6347**
- STADTPULS-440: feat(redis): Use redis for reate limiting
- STADTPULS-440: feat(supabase plugin): Moves supabase plugin into source
- STADTPULS-440: test(k6): Load test application with k6
- STADTPULS-440: chore(envs): Adds needed envs
- STADTPULS-440: chore(esbuild): Remove etsc as dep and use esbuild directly
- STADTPULS-440: ci(redis): Adds redis service container to actions
- STADTPULS-440: housekeeping + lock files

### CODE REVIEW

**Comment by ff6347** [comment]
@vogelino @dnsos Just take a look if zou spot something wired

File: src/lib/server.ts [hunk 1]
```diff
@@ -78,6 +83,8 @@ export const buildServer: (options: {
   server.register(fastifyBlipp);
   server.register(fastifyRateLimit, {
     allowList: ["127.0.0.1"],
+    cache: 5000,
+    redis,
```
Comment (vogelino) [inline src/lib/server.ts:87]: Check if cache makes sense with redis.

File: src/lib/supabase.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        timeout: 1,
+      },
+    }
+  );
+  if (!fastify.supabase) {
+    fastify.decorate("supabase", supabaseAdmin);
+  }
+};
+
+// async function main(): Promise<void> {
```
Comment (vogelino) [inline src/lib/supabase.ts:43]: What is this commented code?

**Review by vogelino — COMMENTED** [review]
LGTM but it isn't my expertise at all. It makes sense though! 👍

File: src/lib/supabase.ts (hunk 2 above)
Comment (ff6347) [reply src/lib/supabase.ts:43]: Thanks. Was a left over from the external repo.

File: src/lib/server.ts (hunk 1 above)
Comment (ff6347) [reply src/lib/server.ts:87]: Thanks. should have no effect on redis, since it is a setting related to lru in memory cache.

**Pull request merged**